web/client/localConfig.json

```diff
@@ -635,7 +635,19 @@
                 "containerPosition": "header",
                 "className": "navbar shadow navbar-home"
             }
-        }, "Login", "Language", "NavMenu", "DashboardSave", "DashboardSaveAs", "Attribution", "Home", {
+        }, "Login", "Language", "NavMenu",
+          {
+            "name": "DashboardSave",
+            "cfg": {
+              "disablePluginIf": "{!!(state('browser') && state('browser').mobile)}"
```

Comment:
disablePluginIf can be placed directly on plugin. Please place it inside the plugin as default.